src/validation.h

```diff
@@ -556,7 +557,7 @@ class CChainState
         size_t cache_size_bytes,
         bool in_memory,
         bool should_wipe,
-        std::string leveldb_name = "chainstate");
+        std::string leveldb_name = CHAINSTATE_DIRNAME);
```

Comment:
Why is this change needed?

Reply:
You are right - it is no longer needed with the `-reindex` change. Removed.